kubernetes/flink-operator/flink-cluster-templates/netflow/mvnNetflowAggregator/flinkoperator-flinkjobcluster-netflow.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: flinkoperator.k8s.io/v1beta1
+kind: FlinkCluster
+metadata:
+  name: flinkjobcluster-netflowaggr-driver-v1
+spec:
+  flinkVersion: "1.14"
+  image:
+    name: kubernetes:31320/netflowaggregator-driver:latest 
```

Comment:
Why does the image URL point to `kubernetes` service?

Reply:
`kubernetes:31320` is the private registry where Images are uploaded, any Flink application is uploaded into the private registry and then used by Kubernetes